frontend/test/metabase/scenarios/dashboard/dashboard_data_permissions.cy.spec.js

```diff
@@ -25,7 +25,7 @@ function filterDashboard(suggests = true) {
 }
 
 describe("support > permissions (metabase#8472)", () => {
-  before(() => {
+  beforeEach(() => {
     restore();
     signIn("admin");
```

Comment:
Could `signInAsAdmin` and `signIn("admin")` be unified?

Reply:
My best guess is that the author was using `signIn()` because it requires only one import for both admin and normal used, compared to two separate imports.

I can change it, no problem.

Reply:
There are several places, not just the two comments I pointed out - you're welcome to split it into a separate PR if easier